.github/workflows/publish.yml

```diff
@@ -1,71 +1,387 @@
-name: "Publish libraries"
+name: "Publish all libs, Arrow-stack and Docs"
 
 on:
   push:
     branches:
-    - main
-    - release/**
+      - main
+      - release/**
     paths:
-    - 'arrow-libs/**'
-    - '.github/**'
+      - 'arrow-libs/**'
+      - '.github/**'
+
+env:
+  BASEDIR: ${{github.workspace}}/arrow-libs
```

Comment:
This is not an env property shared for all jobs, is it?

Reply:
Yes it's used as an env variable for a script